### PR TITLE
enh(update): add known issue

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Add know issue in upgrade procedure

## Target serie

- [ ] 20.04.x
- [x] 20.10.x (master)

### CODE REVIEW

**Comment by cgagnaire** [comment]
Why here and not in the other upgrade procedures ?

File: fr/upgrade/upgrade-from-20-04.md [hunk 1]
```diff
@@ -6,6 +6,30 @@ title: Montée de version depuis Centreon 20.04
 Ce chapitre décrit la procédure de montée de version de votre plate-forme
 Centreon depuis la version 20.04 vers la version 20.10.
 
+> Comportements connus:
+>
+>   - Si Engine sur un Poller ou un Remote Server n'est pas mis à jour en
+>     20.10, les fichiers de configuration copiés depuis un Central à jour
```
Comment (omercier) [inline fr/upgrade/upgrade-from-20-04.md:12]:
```suggestion
>   - Si centreon-engine n'est pas mis à jour en 20.10 sur un Poller ou un Remote
>     Server, les fichiers de configuration copiés depuis un Central à jour
```

File: en/upgrade/upgrade-from-20-04.md [hunk 2]
```diff
@@ -6,6 +6,30 @@ title: Upgrade from Centreon 20.04
 This chapter describes how to upgrade your Centreon platform from version 20.04
 to version 20.10.
 
+Known behaviours:
+>
+>   - If Engine on a Poller or Remote Server is not upgraded to 20.10,
+>     configuration files copied from an upgraded Central by Gorgone using ZMQ
```
Comment (omercier) [inline en/upgrade/upgrade-from-20-04.md:12]:
```suggestion
>   - If centreon-engine is not upgraded to 20.10 on a Poller or Remote Server,
>     configuration files copied from an upgraded Central by Gorgone using ZMQ
```

**Comment by cg-tw** [comment]
This is obsolete

**Pull request closed without merging**